tests/Tests/CommonOptions/AutoExpandReplicas/AutoExpandReplicasTests.cs

```diff
@@ -39,15 +39,15 @@ public class AutoExpandReplicasTests
 		public void ImplicitConversionFromNullString()
 		{
 			string nullString = null;
-			OpenSearch.Client.AutoExpandReplicas autoExpandReplicas = nullString;
+			OpenSearch.Client.Specification.IndicesApi.AutoExpandReplicas autoExpandReplicas = nullString;
```

Comment:
Maybe it is worth adding an import to avoid specifying a long namespace each time?

Reply:
These were due to the namespace the tests are in colliding with the type name they're trying to reference. Fix for this is to bring the `using` declaration inside the namespace declaration so that it takes precedent.